spec/controllers/two_factor_authentication/webauthn_verification_controller_spec.rb

```diff
@@ -100,6 +102,11 @@
         expect(@analytics).to receive(:track_event).
           with('User marked authenticated', authentication_type: :valid_2fa)
 
+          expect(@irs_attempts_api_tracker).to receive(:track_event).with(
+            :mfa_verify_webauthn_platform,
+            success: true,
+          )
```

Comment:
i think we should unindent this to match
```suggestion
        expect(@irs_attempts_api_tracker).to receive(:track_event).with(
          :mfa_verify_webauthn_platform,
          success: true,
        )
```

Reply:
Oops!  Good catch!